drivers/display/display_lpm0xx.c

```diff
@@ -0,0 +1,362 @@
+/*
+ * Copyright (c) 2020 Rohit Gujarathi
```

Comment:
Where does this code originate from?

Reply:
It was originally copied from ls0xx.c file on the same directory.